### PR TITLE
Minor Typo_s

### PR DESCRIPTION
**Pull request by sieis**
A few _super_small typos in the text.

### CODE REVIEW

**Review by kotp — APPROVED** [review]
Good find, sentences in bullets that do not complete the stem for sure need periods.

I like it.  (My approval means nothing, but may speed up an approval and merge, maybe.)

**Comment by kotp** [comment]
If you are going to continue committing as you go, you may want to set this as draft, until you are ready for review.

**Comment by sieis** [comment]
Good call; thanks! Still learning the ins and outs :)

File: Solidity_And_Smart_Contracts/Section_2/Lesson_1_Get_Your_Local_Ethereum_Network_Running.md [hunk 1]
```diff
@@ -13,16 +13,16 @@ So in this case, we want people to 👋 at us. The bigger picture here is:
 
 3\. **We're going to build a client website** that will let people easily interact with our smart contract on the blockchain.
 
-I'll explain certain things in-depth as needed (ex. how mining working, how smart contracts are compiled and ran, etc) *but for now lets just focus on getting stuff running*.
+I'll explain certain things in-depth as needed (ex. how mining working, how smart contracts are compiled and run, etc) *but for now let's just focus on getting stuff running*.
```
Comment (adilanchian) [inline Solidity_And_Smart_Contracts/Section_2/Lesson_1_Get_Your_Local_Ethereum_Network_Running.md:16]: Little miss here:

Can you change
`ex. how mining working, how smart contracts are compiled and run, etc` ->

`ex. how mining works, how smart contracts are compiled and run, etc.`

File: Solidity_And_Smart_Contracts/Section_2/Lesson_3_Compile_Contract_Locally.md [hunk 2]
```diff
@@ -114,4 +114,4 @@ So what does this mean? Well, every time you run a terminal command that starts
 
 Go to #progress and post a screenshot of your terminal with the output.
 
-Be sure to make that console.log whatever you want! You've now written your own contract and ran it by deploying to a local blockchain WOOOOOOOOOO LETS GOOO.
+Be sure to make that console.log whatever you want! You've now written your own contract and run it by deploying to a local blockchain WOOOOOOOOOO LETS GOOO.
```
Comment (adilanchian) [inline Solidity_And_Smart_Contracts/Section_2/Lesson_3_Compile_Contract_Locally.md:117]: This is actually correct and should stay as `ran` :).

Please revert!

File: Solidity_And_Smart_Contracts/Section_2/Lesson_4_Store_Data_On_Smart_Contract.md [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
 
-From here, lets add some fanciness to our contract.
+From here, let's add some fanciness to our contract.
 
 We want to be able to let someone wave at us and then store that wave.
 
 So, first thing we need is a function they can hit to wave at us!
 
-The blockchain = Think of it as a cloud provider, kinda like AWS. But, it's owned by no one. And, is ran through compute power powered by random people called miners. We pay miners to run our code.
+The blockchain = Think of it as a cloud provider, kinda like AWS. But, it's owned by no one. And, is run through compute power powered by random people called miners. We pay miners to run our code.
```
Comment (adilanchian) [inline Solidity_And_Smart_Contracts/Section_2/Lesson_4_Store_Data_On_Smart_Contract.md:10]: Can you change this to say:

```
The blockchain = Think of it as a cloud provider, kinda like AWS, but it's owned by no one. It's ran by compute power from mining machines all over the world. Usually these people are called miners and we pay them to run our code!
```

**Review by adilanchian — CHANGES_REQUESTED** [review]
Added some things to tweak here! Thanks so much for this :)

**Comment by sieis** [comment]
@adilanchian I got those edits and reversions you mentioned fixed now, and addressed the few more typos I found going through the rest of the project.

**Review by adilanchian — APPROVED** [review]

**Comment by adilanchian** [comment]
This is an EPIC change. Thanks for taking the time to add all these changes for us! Merging you in right now :)

**Comment by adilanchian** [comment]
@all-contributors please add @sieis for doc

**Comment by sieis** [comment]
You bet; happy to help, and I appreciate y'all putting this together!

**Pull request merged**